Source/VSIX/AllRelativityTemplates/AllRelativityTemplates.csproj

```diff
@@ -440,6 +440,10 @@
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Include="packages\Newtonsoft.Json.13.0.2.nupkg">
```

Comment:
<Content Include="packages\Newtonsoft.Json.8.0.3.nupkg">
      <IncludeInVSIX>true</IncludeInVSIX>
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </Content>
and 

    <Content Include="packages\Newtonsoft.Json.12.0.2.nupkg">
      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      <IncludeInVSIX>true</IncludeInVSIX>
    </Content>
are in this file as well.. do they need to be listed?

Reply:
Hm, probably not.  I can try to get this cleaned up

Reply:
Done